packages/material-ui/src/MenuItem/MenuItem.test.js

```diff
@@ -31,30 +32,31 @@ describe('<MenuItem />', () => {
   }));
 
   it('should render a button ListItem with with ripple', () => {
-    const wrapper = shallow(<MenuItem />);
-    assert.strictEqual(wrapper.type(), ListItem);
+    const wrapper = mount(<MenuItem />);
```

Comment:
why'd we change these from shallow to mount?

Reply:
Not sure, we should change it back.